includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -194,20 +194,20 @@ public function script_loader_tag( $tag, $handle, $src ): string {
 	 *
 	 * @since 1.6.0
 	 *
-	 * @param string $avatar      HTML for the user's avatar.
-	 * @param mixed  $id_or_email The avatar to retrieve. Accepts a user_id, Gravatar MD5 hash,
-	 *                            user email, WP_User object, WP_Post object, or WP_Comment object.
-	 * @param int    $size        Square avatar width and height in pixels to retrieve.
-	 * @param string $default     URL for the default image or a default type. Accepts '404', 'retro', 'monsterid',
-	 *                            'wavatar', 'indenticon', 'mystery', 'mm', 'mysteryman', 'blank', or
-	 *                            'gravatar_default'. Default is the value of the 'avatar_default' option, with a
-	 *                            fallback of 'mystery'.
-	 * @param string $alt         Alternative text to use in the avatar image tag. Default empty.
-	 * @param array  $args        Arguments passed to get_avatar_data(), after processing.
+	 * @param string|mixed $avatar      HTML for the user's avatar.
+	 * @param mixed        $id_or_email The avatar to retrieve. Accepts a user_id, Gravatar MD5 hash,
+	 *                                  user email, WP_User object, WP_Post object, or WP_Comment object.
+	 * @param int          $size        Square avatar width and height in pixels to retrieve.
+	 * @param string       $default     URL for the default image or a default type. Accepts '404', 'retro', 'monsterid',
+	 *                                  'wavatar', 'indenticon', 'mystery', 'mm', 'mysteryman', 'blank', or
+	 *                                  'gravatar_default'. Default is the value of the 'avatar_default' option, with a
+	 *                                  fallback of 'mystery'.
+	 * @param string       $alt         Alternative text to use in the avatar image tag. Default empty.
+	 * @param array        $args        Arguments passed to get_avatar_data(), after processing.
 	 *
-	 * @return string
+	 * @return string|mixed
 	 */
-	public function get_avatar( $avatar, $id_or_email, $size, $default, $alt, array $args ): string {
+	public function get_avatar( $avatar, $id_or_email, $size, $default, $alt, array $args ) {
```

Comment:
If we retain the return type in `add_attribute()`, we could also retain it here.

Reply:
As noted before, `add_attribute` doesn't always return a string